packages/tools/devtools/devtools-core/src/CommonInterfaces.ts

```diff
@@ -45,3 +48,41 @@ export interface HasFluidObjectId {
 	 */
 	fluidObjectId: FluidObjectId;
 }
+
+/**
+ * Base interface used in message data for edits allowed by a particular Fluid object (DDS) via
+ * an enum.
+ *
+ * @public
+ */
+export enum EditType {
+	number = "number",
```

Comment:
Nit: We generally use `PascalCase` for enum flag names. E.g. `Number = "number",`.